flaky test TestLocalTempKVDirCheckBasic

## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)

<!-- a step by step guide for reproducing the bug. -->

### 2. What did you expect to see? (Required)

### 3. What did you see instead (Required)
The test branch is the master branch, and Go 1.25 is used. （https://do.pingcap.net/jenkins/job/pingcap/job/tidb/job/pull_unit_test_next_gen/6689/）
```
precheck_impl_test.go:413: check result message: local disk resources are rich, estimate sorted data size 3.032TiB, local available is 3.189TiB
    precheck_impl_test.go:414: 
        	Error Trace:	lightning/pkg/importer/precheck_impl_test.go:414
        	Error:      	Should be false
        	Test:       	TestPrecheckImplSuite/TestLocalTempKVDirCheckBasic
--- FAIL: TestPrecheckImplSuite/TestLocalTempKVDirCheckBasic (0.00s)
```
### 4. What is your TiDB version? (Required)

<!-- Paste the output of SELECT tidb_version() -->

--- a/lightning/pkg/importer/BUILD.bazel
+++ b/lightning/pkg/importer/BUILD.bazel
@@ -132,6 +132,7 @@ go_test(
         "//lightning/pkg/importer/opts",
         "//lightning/pkg/precheck",
         "//lightning/pkg/web",
+        "//pkg/config/kerneltype",
         "//pkg/ddl",
         "//pkg/errno",
         "//pkg/kv",

--- a/lightning/pkg/importer/precheck_impl_test.go
+++ b/lightning/pkg/importer/precheck_impl_test.go
@@ -26,6 +26,7 @@ import (
 	"github.com/pingcap/tidb/lightning/pkg/importer/mock"
 	ropts "github.com/pingcap/tidb/lightning/pkg/importer/opts"
 	"github.com/pingcap/tidb/lightning/pkg/precheck"
+	"github.com/pingcap/tidb/pkg/config/kerneltype"
 	"github.com/pingcap/tidb/pkg/lightning/checkpoints"
 	"github.com/pingcap/tidb/pkg/lightning/config"
 	"github.com/pingcap/tidb/pkg/lightning/log"
@@ -375,6 +376,9 @@ func (s *precheckImplSuite) TestLocalDiskPlacementCheckBasic() {
 }
 
 func (s *precheckImplSuite) TestLocalTempKVDirCheckBasic() {
+	if kerneltype.IsNextGen() {
+		s.T().Skip("we only support global sort in nextgen kernel, this is for local-sort")
+	}
 	var (
 		err    error
 		ci     precheck.Checker

